colosseum/constants.py

```diff
@@ -45,6 +48,41 @@ def __str__(self):
         return ", ".join(sorted(choices))
 
 
+class OtherProperty:
+    """
+    A class to refer to a callable to perform operations with other properties.
+
+    If `name` is provided, the value of that property will be used.
+
+    If a more complex operation is required, subclass and override the `value` method.
+    """
+
+    def __init__(self, name=None):
+        if name is not None and not isinstance(name, str):
+            raise TypeError('`name` must be a valid CSS property string!')
```

Comment:
If there's a need for a base abstract class, then lets make it one, rather than trying to implement this kind of error protection logic.

Reply:
Nahhh no need, I simplified this code

Reply:
This type check isn't strictly needed... and it's a weak check anyway - we're not checking if it *is* a valid CSS property name, and it's not public API anyway (since OtherProperty is only used as part of the declaration process).